test/dummy/app/views/icons/show.html.erb

```diff
@@ -10,5 +10,3 @@ Hello there
     <%= image_tag currency.path, id: "#{currency.name}" %>
   <% end %>
 </div>
-
-<%= image_tag "chocolate.png", id: "chocolate" %>
```

Comment:
Because it doesn't do anything and it was causing deprecations. (Assets `chocolate.png` is not part of the asset pipeline, Sprockets 3.2 deprecated that behaviour)